src/Data/Bases/flask.lua

```diff
@@ -356,9 +356,9 @@ itemBases["Sulphur Flask"] = {
 itemBases["Silver Flask"] = {
 	type = "Flask",
 	subType = "Utility",
-	tags = { flask = true, utility_flask = true, not_for_sale = true, no_effect_flask_mod = true, default = true, },
+	tags = { flask = true, utility_flask = true, not_for_sale = true, default = true, },
 	implicitModTypes = { },
-	flask = { duration = 6, chargesUsed = 40, chargesMax = 60, buff = { "Onslaught" }, },
+	flask = { duration = 6, chargesUsed = 40, chargesMax = 60, buff = { "" }, },
```

Comment:
I don't get why this was removed from the data, and until we can figure out where it lives now, we should exclude it from the export.